docs/_docs/maintaining/releasing-a-new-version.md

```diff
@@ -69,6 +69,14 @@ And then, you're done! :tada: Feel free to celebrate!
 
 If you have access to the [@jekyllrb](https://twitter.com/jekyllrb) Twitter account, you should tweet the release post from there. If not, just ask another maintainer to do it or to give you access.
 
+### Build the docs
+
+We package our documentation as a :gem:Gem for offline use.
```

Comment:
":gem: Gem" is missing a space, I think?

Reply:
Yeah, I suppose it’s not all one word.